plugins/inventory/aws_mq.py

```diff
@@ -0,0 +1,286 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+name: aws_mq
+short_description: MQ broker inventory source
+description:
+  - Get brokers from Amazon Web Services MQ.
+  - Uses a YAML configuration file that ends with aws_mq.(yml|yaml).
+options:
+  regions:
+    description:
+      - A list of regions in which to describe MQ brokers. Available regions are listed here
+        U(https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/)
+    type: list
+    elements: str
+    default: []
+  strict_permissions:
+    description: By default if an AccessDenied exception is encountered this plugin will fail. You can set strict_permissions to
+      False in the inventory config file which will allow the restrictions to be gracefully skipped.
+    type: bool
+    default: True
+  statuses:
+    description:
+      - A list of desired states for brokers to be added to inventory. Set to ['all'] as a shorthand to find everything.
+        Possible value are listed here U(https://docs.aws.amazon.com/amazon-mq/latest/developer-guide/broker-statuses.html)
+    type: list
+    elements: str
+    default:
+      - RUNNING
+      - CREATION_IN_PROGRESS
+  hostvars_prefix:
+    description:
+      - The prefix for host variables names coming from AWS.
+    type: str
+  hostvars_suffix:
+    description:
+      - The suffix for host variables names coming from AWS.
+    type: str
+notes:
+  - Ansible versions prior to 2.10 should use the fully qualified plugin name 'amazon.aws.aws_mq'.
```

Comment:
```suggestion
  - Ansible versions prior to 2.10 should use the fully qualified plugin name 'community.aws.aws_mq'.
```

Reply:
Do we need this comment?  We've dropped support for Ansible Core < 2.12 (6.0.0)

Reply:
I wasn't sure! But if you confirm, then it's ok to drop it.

Reply:
`AWSInventoryBase` won't work with Ansible < 2.12, so I don't think there's a need for this note.  (AnsiblePlugin.get_options was added in 2.12)

Reply:
addressed